runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/jobsubmission/InMemoryJobService.java

```diff
@@ -70,26 +71,43 @@
   public static InMemoryJobService create(
       Endpoints.ApiServiceDescriptor stagingServiceDescriptor,
       Function<String, String> stagingServiceTokenProvider,
+      ThrowingConsumer<String> cleanupJobFn,
+      Boolean cleanArtifactsPerJob,
```

Comment:
We don't need the boolean. We can simply set the jobTerminationListener in previous line or not set it.